bundles/org.openhab.binding.homematic/src/main/resources/homematic/extra-descriptions_de.properties

```diff
@@ -19,104 +19,105 @@ BC-SC-Rd-WM-2=MAX! Fensterkontakt
 HMIP-PS=Homematic IP Schaltsteckdose
 HMIP-PSM=Homematic IP Schalt-Mess-Steckdose
-HMIP-eTRV=Homematic IP Heizk�rperthermostat
-HMIP-eTRV-2=Homematic IP Heizk�rperthermostat
-HMIP-eTRV-B=Homematic IP Heizk�rperthermostat Basis
-HMIP-eTRV-C=Homematic IP Heizk�rperthermostat kompakt
-HmIP-SMI=Homematic IP Bewegungsmelder mit D�mmerungssensor
-HmIP-SMO=Homematic IP Bewegungsmelder mit D�mmerungssensor au�en
-HmIP-KRC4=Homematic IP Schl�sselbundfernbedienung - 4 Tasten
+HMIP-eTRV=Homematic IP Heizkörperthermostat
+HMIP-eTRV-2=Homematic IP Heizkörperthermostat
+HMIP-eTRV-B=Homematic IP Heizkörperthermostat Basis
+HMIP-eTRV-C=Homematic IP Heizkörperthermostat kompakt
+HmIP-SMI=Homematic IP Bewegungsmelder mit Dämmerungssensor
+HmIP-SMO=Homematic IP Bewegungsmelder mit Dämmerungssensor außen
+HmIP-KRC4=Homematic IP Schlüsselbundfernbedienung - 4 Tasten
 HmIP-RC8=Homematic IP Fernbedienung - 8 Tasten
-HmIP-DRA=Homematic IP Hutschienenadapter f�r Multi IO Box
+HmIP-DRA=Homematic IP Hutschienenadapter für Multi IO Box
 HmIP-WRC6=Homematic IP Wandtaster - 6-fach
-HmIP-FAL24-TR=Homematic IP Trafo f�r Fu�bodenheizungsaktoren - 24 V
-HmIP-FAL24-C6=Homematic IP Fu�bodenheizungsaktor - 6-fach, 24 V
-HmIP-FAL24-C10=Homematic IP Fu�bodenheizungsaktor - 6-fach, 24 V
+HmIP-FAL24-TR=Homematic IP Trafo für Fußbodenheizungsaktoren - 24 V
+HmIP-FAL24-C6=Homematic IP Fußbodenheizungsaktor - 6-fach, 24 V
+HmIP-FAL24-C10=Homematic IP Fußbodenheizungsaktor - 6-fach, 24 V
 HmIP-SRH=Homematic IP Fenstergriffsensor
 HmIP-WTH-2=Homematic IP Wandthermostat mit Luftfeuchtigkeitssensor
-HmIP-FAL230-C6=Homematic IP Fu�bodenheizungsaktor - 6-fach, 230 V
-HmIP-FAL230-C10=Homematic IP Fu�bodenheizungsaktor - 6-fach, 230 V
-HmIP-BDT=Homematic IP Dimmaktor f�r Markenschalter
-HmIP-BSM=Homematic IP Schalt-Mess-Aktor f�r Markenschalter
+HmIP-FAL230-C6=Homematic IP Fußbodenheizungsaktor - 6-fach, 230 V
+HmIP-FAL230-C10=Homematic IP Fußbodenheizungsaktor - 6-fach, 230 V
+HmIP-BDT=Homematic IP Dimmaktor für Markenschalter
+HmIP-BSM=Homematic IP Schalt-Mess-Aktor für Markenschalter
 HmIP-FSM=Homematic IP Schalt-Mess-Aktor - Unterputz
 HmIP-SWSD=Homematic IP Rauchwarnmelder
 HmIP-ASIR=Homematic IP Alarmsirene
 HmIP-PDT-UK=Dimmer-Steckdose - Phasenabschnitt (UK-Version)
-HmIP-eTRV-UK=Heizk�rperthermostat (UK-Version)
-HmIP-SWDO-I=Fenster- und T�rkontakt - verdeckten Einbau
+HmIP-eTRV-UK=Heizkörperthermostat (UK-Version)
+HmIP-SWDO-I=Fenster- und Türkontakt - verdeckten Einbau
 HmIP-BWTH=Wandthermostat mit Schaltausgang
 HmIP-BWTH24=Wandthermostat mit Schaltausgang
-HmIP-PCBS-BAT=Schaltplatine f�r Batteriebetrieb
+HmIP-PCBS-BAT=Schaltplatine für Batteriebetrieb
 HmIP-FROLL=Homematic IP Rolladenaktor
-HmIP-BROLL=Homematic IP Rolladenaktor f�r Markenschalter
-HmIP-BRC2=Homematic IP Wandtaster f�r Markenschalter - 2-fach
+HmIP-BROLL=Homematic IP Rolladenaktor für Markenschalter
+HmIP-BRC2=Homematic IP Wandtaster für Markenschalter - 2-fach
+HmIP-MOD-Homematic IP Tormatic Modul
```

Comment:
What are your changes? to this file only this? It seems that the equals misses here.

Reply:
I added the "HmIP-MOD-Homematic IP Tormatic Modul". Its the only changed line. I don´t know why the GitHub-Web-Editor marked the other Lines as changed.

Reply:
Do you agree with the DCO? See https://community.openhab.org/t/dco-check-signing-off-with-github-web-editor-explanation/83330

If so then I can recreate your commit and fix it using a normal PR.

Reply:
Yes, this would be great